gcsio/pom.xml

```diff
@@ -93,6 +93,14 @@
       <groupId>io.grpc</groupId>
       <artifactId>grpc-protobuf</artifactId>
     </dependency>
+    <dependency>
+      <groupId>com.google.api.grpc</groupId>
+      <artifactId>proto-google-cloud-storage-v2</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>com.google.api.grpc</groupId>
+      <artifactId>grpc-google-cloud-storage-v2</artifactId>
```

Comment:
What this dependency brings in in addition to `proto-google-cloud-storage-v2`?

Reply:
`grpc-google-cloud-storage-v2` has the `StorageGrpc` class with APIs, removing `proto-google-cloud-storage-v2` as it is a dependency from `grpc-google-cloud-storage-v2` anyways